lib/chai/assertion.js

```diff
@@ -138,11 +138,20 @@ module.exports = function (_chai, util) {
     if (!ok) {
       msg = util.getMessage(this, arguments);
       var actual = util.getActual(this, arguments);
-      throw new AssertionError(msg, {
-          actual: actual
+      var assertionErrorObjectProperties = {
+        actual: actual
         , expected: expected
         , showDiff: showDiff
-      }, (config.includeStack) ? this.assert : flag(this, 'ssfi'));
+      };
+
+      var operator = util.getOperator(this, arguments);
+      if (operator) {
+        assertionErrorObjectProperties.operator = operator;
+      }
+
+      throw new AssertionError(msg,
```

Comment:
Can we move `msg` to the next line so that we can have 1 argument per line? (Nitpicking, sorry)